### DISCUSSION TITLE
[Feature Request] Allow multiple default recipients

### DISCUSSION THREAD

**Discussion by fotomut** [anonaddy/anonaddy#364] [Feature Requests]
Thanks so much for this incredible product. It has truly improved the efficiency and safety of our online interactions. 

I would like to be able to set one of my custom domains to always default to two recipients. The use-case is that I have one domain that is mostly for me, one mostly for my spouse, and the third domain we mostly share. It's annoying to have to manually set two recipients for every single email I want on that third domain. Or if someone misspells an email to that domain, it only goes to me and my spouse doesn't see it.

Thoughts on how to accomplish this?

**Comment by willbrowningme** [comment]
Thanks for the feedback! I do agree that is a nice idea, I'll add it to the roadmap.

In the meantime you can achieve this by using the recipient's "key" that can be found on the [recipients](https://app.anonaddy.com/recipients) page in the second column.

If you get the keys of the two recipients you want to receive any emails (e.g. 1 and 4) then you can do the following when creating an email for the first time (using catch-all):

alias+1.4@yourdomain.com

This will automatically attach the recpients with keys 1 and 4 to your alias. For more details see this [blog post](https://anonaddy.com/blog/add-recipients-to-aliases-at-creation-and-new-uuid-aliases/).

**Reply by jdrehobl** [reply]
@willbrowningme any updates on this. My spouse and I have the same exact use case as @fotomut. While specifying the keys is a workaround, being able to set two recipients per domain would be a great improvement. 

I see [this](https://github.com/orgs/anonaddy/projects/1?pane=issue&itemId=63479154) on the roadmap (as a draft) that looks similar. Is that for what @fotomut is requesting here? 

Thanks!

**Reply by willbrowningme** [reply]
This can currently be achieved by creating a new [https://app.addy.io/rules](rule) that matches all alias emails ending with your custom domain e.g. "@example.com" and then set the two actions to "Forward To" each of the recipients you want.

**Reply by jdrehobl** [reply]
that works - thanks!
